app/scripts/lib/util/chrome-util.js

```diff
@@ -35,7 +42,17 @@ module.exports = {
 
   syncGetValue(key) {
     return promised((cb) => {
-      chrome.storage.sync.get(key, (items) => cb(items.hasOwnProperty(key) ? items[key] : undefined));
+      chrome.storage.sync.get(key, (items) => {
+        try {
+          cb(items.hasOwnProperty(key) ? items[key] : undefined);
+        } catch (err) {
+          // Firefox causes an security error when storage.local is accessed from popup scripts.
+          // https://bugzilla.mozilla.org/show_bug.cgi?id=1258139
+          // Perhaps this hack can be removed when storage.sync is officially supported, or the above bug is resolved.
```

Comment:
It failed. :cry: 

https://bugzilla.mozilla.org/show_bug.cgi?id=1258139#c1 implies the storage access requires some privileges and popup scripts don't have the privileges.
So IMO, even if we use `getBackgroundPage` to call a background script's method it's called in a "popup script context", which doesn't have a right to access the local storage...

It might resolve this issue to fetch popup settings via postMessage() or something, but if we do so, the code will become complicated :/

Is it better to wait until bug 1258139 is fixed?